tasks/05_participants/task.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+from tqdm import tqdm
+import participants
+from common import db_config, sql
+import time
+import logging
+
+
+# set up a log for diagnostics/debugging
+logging.basicConfig(
+    filename="participants.log", filemode="a", encoding="utf-8", level=logging.INFO
+)
+
+
+def main():
+    participants_query = """
+    SELECT c.id as case_id, c.case_number, c.participants_raw, e.participants_parse_error, p.raw_text
```

Comment:
putting a note in after our conversation today: we probably only need the `raw_text` column from the `pages` column, so references to the `cases` table and `participants_raw` column can be removed

Reply:
I ended up changing this query. Instead of referencing the `cases` table, it now selects only from the `pages` table where the html in the `raw_text` column contains a participants table element.